frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { RootState } from '../store/types'
+import { PATHS } from '../routes/Routes'
+import { DOMAIN_URL, ENDPOINTS } from '../store/endpoints'
+import { months, days } from '../common/dates'
+
+const BookingGroup = styled.div``
+const BookingContainer = styled.div`
+  position: relative;
+  cursor: pointer;
+  background-color: #ffffff;
+  margin: 23px;
+  min-height: 70px;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  padding: 15px;
+`
+const BookingLeftDisplay = styled.div`
+  align-self: center;
+  width: 80%;
+`
+
+const DateLabel = styled.div`
+  margin-bottom: 0em;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  overflow: hidden;
+  color: #ff705c;
+`
+
+const TimeLabel = styled.div`
```

Comment:
should this be using label instead of div?

Reply:
Resolved in commit 9c59d802492bb5f95b587c34ae11e07f5965c73a. Should be a div, changed naming of component.